bbn-test-5/babylon-node/README.md

```diff
@@ -0,0 +1,269 @@
+
+1. Install [Golang 1.21](https://go.dev/dl)
+2. Verify installation:
+
+```shell
+go version
+```
+
+3. Clone and build Babylon:
+```shell
+git clone git@github.com:babylonlabs-io/babylon.git
+cd babylon
+git checkout bbn-test-5  
+make install
+```
+<!-- TODO: testnet tag to be defined -->
+This command does the following:
+- Builds the daemon
+- Compiles all the Go packages in the project
+- Installs the binary 
+- Makes the `babylond` command globally accessible from your terminal
+
+You should expect to see the following output:
+
+```shell
+# Build output will show:
+go install -mod=readonly -tags "netgo ledger mainnet" \
+    -ldflags '\
+        -X github.com/cosmos/cosmos-sdk/version.Name=babylon \
+        -X github.com/cosmos/cosmos-sdk/version.AppName=babylond \
+        -X github.com/cosmos/cosmos-sdk/version.Version=v0.13.0 \
+        -X github.com/cosmos/cosmos-sdk/version.
+        Commit=976e94b926dcf287cb487e8f35dbf400c7d930cc \
+        -X "github.com/cosmos/cosmos-sdk/version.BuildTags=netgo,ledger" \
+        -w -s' \
+    -trimpath ./...
+```
+
+Now that the binary has been successfully installed, 
+let's check the available actions through the `babylond` command:
+
+```shell
+babylond
+Available Commands:
+  add-genesis-account Add a genesis account to genesis.json
+  collect-gentxs      Collect genesis txs and output a genesis.json file
+  comet               CometBFT subcommands
+  config              Utilities for managing application configuration
+  ...
+```
+
+If the `babylond` command isn't found, ensure `$GOPATH/bin` is in your shell's 
+`$PATH`. Add it with:
+
+ ```shell 
+ echo 'export PATH=$HOME/go/bin:$PATH' >> ~/.profile 
+ ```
+
+## Setup your node, home directory, and configuration
+
+Initialize your node and create the necessary configuration directory. 
+This command will generate several important configuration files 
+including `app.toml`, `client.toml`, and `genesis.json`:
+
+```shell
+babylond init <moniker> --chain-id bbn-test-5 --home <path>
+```
+
+The `<moniker>` is a unique identifier for your node (e.g. `node0`).
+The `--home` flag specifies the directory where your node files will be stored (e.g. `--home ./nodeDir`).
+
+After initialization, you'll need to modify two important configuration files:
+
+1. First, open `app.toml` and update these essential settings from what is auto-generated:
+
+```shell
+# Base configuration
+minimum-gas-prices = "0.005ubbn"
+iavl-cache-size = 0
+iavl-disable-fastnode=true
+
+[btc-config]
+
+# Configures which bitcoin network should be used for checkpointing
+# valid values are: [mainnet, testnet, simnet, signet, regtest]
+network = "signet" # The Babylon testnet connects to the signet Bitcoin network
+```
+
+Navigate to `config.toml`. Add in your seed, as shown below:
+
+```shell
+ #P2P Configuration Options    
+
+# Comma separated list of seed nodes to connect to
+seeds = "8fa2d1ab10dfd99a51703ba760f0ef555ae88f36@16.162.207.201:26656"
```

Comment:
This will not be the endpoint of the testnet seed.
Here, we will point to Nodes.Guru seed node or to a page listing several seed nodes in the networks repository. Atm, I'd put some placeholder.